dvc/repo/reproduce.py

```diff
@@ -58,7 +61,7 @@ def reproduce(
     all_pipelines=False,
     **kwargs
 ):
-    from dvc.stage import Stage
+    from ..dvcfile import Dvcfile
```

Comment:
Already imported above. Better to keep these import dynamic, same as they were before.